src/main/java/com/rickmorty/Controllers/FavoriteController.java

```diff
@@ -40,12 +63,30 @@ public ResponseEntity<Page<FavoriteResponseDto>> getAllFavorites(
         return ResponseEntity.ok(favorites);
     }
 
+    @Operation(summary = "Remove a specific favorite for a user",
+            description = "Remove a specific favorite for a user by user ID and favorite ID",
+            responses = {
+                    @ApiResponse(responseCode = "204", description = "Favorite removed"),
+                    @ApiResponse(responseCode = "404", description = "Favorite not found", 
+                                 content = @Content(mediaType = "application/json", 
+                                                    examples = @ExampleObject(value = "{\"message\": \"Favorito não encontrado\"}"))),
+            })
     @DeleteMapping("/{userId}/{favoriteId}")
     public ResponseEntity<String> removeFavorite(@PathVariable Long userId, @PathVariable Long favoriteId) {
         favoriteService.removeFavorite(userId, favoriteId);
         return new ResponseEntity<>("Favorito removido com sucesso.", HttpStatus.NO_CONTENT);
     }
 
+    @Operation(summary = "Remove all favorites for a user",
+            description = "Remove all favorites for a user by user ID",
+            responses = {
+                    @ApiResponse(responseCode = "204", description = "All favorites removed",
+                    content = @Content(mediaType = "application/json", 
+                                                    examples = @ExampleObject(value = "{\"message\": \"Todos os favoritos do usuário foram removidos com sucesso.\"}"))),
```

Comment:
faltando cenário de bad request ao o usuário enviar userId ou favoriteId inválido por exemplo.
Também pode ter cenário que o usuário não existe, na mensagem só tem pra favorite not found